packages/terra-form/src/Textarea.jsx

```diff
@@ -1,18 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-
 import 'terra-base/lib/baseStyles';
+
+import Utilities from './_Utilities';
 import styles from './Textarea.scss';
 
 const cx = classNames.bind(styles);
 
+const TEXTAREA_ROW_SIZES = {
+  small: 2,
+  medium: 5,
+  large: 10,
+};
+
 const propTypes = {
   /**
    * The defaultValue of the textarea. Use this to create an uncontrolled textarea.
    */
   defaultValue: PropTypes.string,
   /**
+   * Whether the textarea can be auto-resized horizontally.
```

Comment:
Does this resize vertically?

Reply:
^ If its isn't vertically resizable, the prop `isAutoResizable` will probably be confusing to consumers

Reply:
It is vertically.

https://github.com/cerner/terra-core/pull/966/commits/60840265df24c9171bb2a185a61ae08580f34064